### PR TITLE
README: View SBX data in local

### PR DESCRIPTION
**Pull request by claudiovallejo**
Added a new section to `README.md` that explains how to sync Sandbox data to your local

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -245,6 +245,17 @@ Run the following command from the project root to create a new dashboard admin:
 bundle exec rails 'create_admin_user[<name>,<email>,<password>]'
 ```
 
+### View Sandbox data in your local
+1. Create a new file in the [ansible/roles/common/files/ssh_keys/sandbox/<your-name>](https://github.com/simpledotorg/deployment/tree/master/ansible/roles/common/files/ssh_keys/sandbox) in the [deployment](https://github.com/simpledotorg/deployment) repo
+2. Run `pbcopy < ~/.ssh/id_rsa.pub` to copy your SSH key
+3. Paste the SSH key in the new file you created
```
Comment (rsanheim) [inline README.md:251]: Can you pull the SSH specific stuff out of here and maybe throw it in a doc or HOWTO in the deployment repo? We could then link to it from here.

File: README.md [hunk 2]
```diff
@@ -245,6 +245,17 @@ Run the following command from the project root to create a new dashboard admin:
...
+4. Create a new branch and submit a PR
+5. Wait for the PR to be reviewed, merged, and deployed to Sandbox
+6. Run `ssh deploy@ec2-13-235-33-14.ap-south-1.compute.amazonaws.com`
+7. Run `bundle exec cap sandbox db:pull` to sync the data with your local machine
+8. Request to be added as a Dashboard Admin to Sandbox
```
Comment (rsanheim) [inline README.md:256]: You would want this to happen before syncing the data, right?

Comment (claudiovallejo) [reply README.md]: I put the steps to request sandbox access before running the commands to sync data. That's what you meant right?

File: README.md [hunk 3]
```diff
@@ -245,6 +245,13 @@ Run the following command from the project root to create a new dashboard admin:
 bundle exec rails 'create_admin_user[<name>,<email>,<password>]'
 ```
 
+### View Sandbox data in your local
+1. Follow the steps in the "How to add an SSH key..." section [here](https://github.com/simpledotorg/deployment) to add your SSH key to the deployment repo
+2. Ask someone from the Simple team to add you as an admin to sandbox
```
Comment (harimohanraj89) [inline README.md:250]: "to the Sandbox dashboard"? To be very clear what kind of admin :)

File: README.md [hunk 4]
```diff
@@ -245,6 +245,13 @@ Run the following command from the project root to create a new dashboard admin:
...
+3. Create a password for your Sandbox account and use that to log into the Dashboard on localhost:3000
```
Comment (harimohanraj89) [inline README.md:251]: You won't be able to do this on `localhost:3000` until after step 5. Should this `on https://api-sandbox.simple.org` instead?

File: README.md [hunk 5]
```diff
@@ -245,6 +245,13 @@ Run the following command from the project root to create a new dashboard admin:
...
+4. Run `ssh deploy@ec2-13-235-33-14.ap-south-1.compute.amazonaws.com`
```
Comment (harimohanraj89) [inline README.md:252]: This is just to test that step 1 worked. We could add a note about that.

```
Run `ssh deploy@ec2-13-235-33-14.ap-south-1.compute.amazonaws.com` to ensure your SSH access from step 1 is working.
```

File: README.md [hunk 6]
```diff
@@ -245,6 +245,13 @@ Run the following command from the project root to create a new dashboard admin:
...
+5. Run `bundle exec cap sandbox db:pull` to sync the data with your local machine
```
Comment (harimohanraj89) [inline README.md:253]: And we can add step 6 if you want here.

```
Start up your server and log into the dashboard at `localhost:3000` using your Sandbox username and password.
```

**Review by harimohanraj89 — APPROVED** [review]
Left some more comments around some nitty gritties. Looks good to go after that!

**Pull request merged**